main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -62,43 +62,50 @@ DataTableCellUI.prototype._render = function() {
   var self = this;
   var cell = this._cell;
 
-  var divContent = $('<div/>')
-  .addClass("data-table-cell-content");
+  var divContent = document.createElement('div');
+  divContent.className = 'data-table-cell-content';
 
-  var editLink = $('<a href="javascript:{}">&nbsp;</a>')
-  .addClass("data-table-cell-edit")
-  .attr("title", $.i18n('core-views/edit-cell'))
-  .appendTo(divContent)
-  .click(function() { self._startEdit(this); });
+  var editLink = document.createElement('a');
+  editLink.className = 'data-table-cell-edit';
+  editLink.setAttribute('title', $.i18n('core-views/edit-cell'));
+  editLink.href = 'javascript:{}';
+  divContent.appendChild(editLink).appendChild(document.createTextNode('\u00A0'));
+  editLink.addEventListener('click', function() { self._startEdit(this); });
 
   $(this._td).empty()
   .unbind()
-  .mouseenter(function() { editLink.css("visibility", "visible"); })
-  .mouseleave(function() { editLink.css("visibility", "hidden"); });
+  .mouseenter(function() { editLink.style.visibility = "visible" })
+  .mouseleave(function() { editLink.style.visibility = "hidden" });
 
   if (!cell || ("v" in cell && cell.v === null)) {
-    $('<span>').addClass("data-table-null").html('null').appendTo(divContent);
+    var nullSpan = document.createElement('span');
+    nullSpan.className = 'data-table-null';
+    nullSpan.innerHTML = 'null';
+    divContent.appendChild(nullSpan);
   } else if ("e" in cell) {
-    $('<span>').addClass("data-table-error").text(cell.e).appendTo(divContent);
+    var errorSpan = document.createElement('span');
+    errorSpan.className = 'data-table-error';
+    errorSpan.textContent = cell.e;
+    divContent.appendChild(errorSpan);
   } else if (!("r" in cell) || !cell.r) {
     if (typeof cell.v !== "string" || "t" in cell) {
       if (typeof cell.v == "number") {
         divContent.addClass("data-table-cell-content-numeric");
       }
-      $('<span>')
-      .addClass("data-table-value-nonstring")
-      .text(cell.v)
-      .appendTo(divContent);
+      var nonstringSpan = document.createElement('span');
+      nonstringSpan.className = 'data-table-value-nonstring';
+      nonstringSpan.textContent = cell.v;
+      divContent.appendChild(nonstringSpan);
     } else if (URL.looksLikeUrl(cell.v)) {
-      $('<a>')
-      .text(cell.v)
-      .attr("href", cell.v)
-      .attr("target", "_blank")
-      .appendTo(divContent);
+      var url = document.createElement('a');
+      url.textContent = cell.v;
+      url.setAttribute('href', cell.v);
```

Comment:
This is used in cases of url so that what's displayed within the cell is the url and the hyperlink for  the `<a>` tag is also the url. It doesn't work without either.